ui/README.md

```diff
@@ -27,12 +46,23 @@ REACT_APP_MAAS_URL="http://<your-local-maas-ip>:5240/MAAS"
 REACT_APP_WEBSOCKET_URL="ws://<your-local-maas-ip>:5240/MAAS/ws"
 ```
 
+## Install Docker inside your container (LXD and multipass)
+
+The simplest way is to use [the convenience script](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script). Once the script has run you will need to add your user to the docker group:
+
+```
+sudo usermod -aG docker $USER
+newgrp docker
+```
```

Comment:
I don't recall ever doing this, are you sure `./run` doesn't do this itself?

Reply:
I can confirm I had to do this when setting up docker in the lxc container (running bionic).

Reply:
I had to do it and so did @lilyvidenova. I've also always had to do it when setting up a new laptop and running any of the other projects that use `./run`